src/types.ts

```diff
@@ -36,3 +36,16 @@ export type SgidClientParams = {
   redirectUri?: string
   hostname?: string
 }
+
+export type RulesParams = {
+  clientId: string
+  accessToken: string
```

Comment:
by right accessToken is enough to derive clientId. It should not be an input. 
To be removed after we implement the auth properly.

Reply:
Removed clientId as a param in [6e9e0ca](https://github.com/opengovsg/sgid-client/pull/65/commits/6e9e0ca1445f1e14c59c7364dc0f9e9a382e8623).